### PR TITLE
feat: Make get work on IPv6 network

### PR DESCRIPTION
**Pull request by flub**

### CODE REVIEW

File: src/get.rs [hunk 1]
```diff
@@ -53,7 +53,11 @@ async fn setup(opts: Options) -> Result<quinn::Connection> {
 
     let tls_client_config = tls::make_client_config(&keypair, opts.peer_id, opts.keylog)?;
     let mut client_config = quinn::ClientConfig::new(Arc::new(tls_client_config));
-    let mut endpoint = quinn::Endpoint::client("0.0.0.0:0".parse().unwrap())?;
+    let bind_addr = match opts.addr.is_ipv6() {
```
Comment (dignifiedquire) [inline src/get.rs:56]: shouldn't we do this on the provide side as well?

Comment (dignifiedquire) [reply src/get.rs:56]: Actually nvm, this is only needed here.

**Review by dignifiedquire — APPROVED** [review]
lgtm, lets wait for @b5 to confirm this works for him

File: src/get.rs (hunk 1 above)
Comment (flub) [reply src/get.rs:56]: the provider or server endpoint already binds to whatever address you give on the commandline, so we already do.  it's not like we need to call a different api for quinn, it's just that we need to have the socket bound to the same family as the server endpoint.

**Pull request merged**